README.md

```diff
@@ -1207,6 +1213,13 @@ General Queries
 /tempZone/home/rods/manager/user_manager.pyc id=212658 size=5233
 ```
 
+Optionally we could query on all but a few of the combined columns from both models.  The columns to be skipped
+can be signalled for omission from the results by negating them:
```

Comment:
I think the second sentence can be simplified to the following:
```
Columns can be excluded from the results by negating them:
```

Reply:
Ok that is done, with a slight little extra added - see if it reads ok

Reply:
Works for me.